scipy/linalg/flapack_gen.pyf.src

```diff
@@ -221,6 +221,41 @@ subroutine <prefix>getrs(n,nrhs,lu,piv,b,info,trans)
 
 end subroutine <prefix>getrs
 
+subroutine <prefix>getc2(n,a,ipiv,jpiv,info)
+    ! lu,ipiv,jpiv,info = getc2(a,overwrite_a=0)
+    ! Compute an LU factorization of with complete pivoting of a general n-by-n matrix.
+    ! A = P * L * U * Q
+    threadsafe
+    callstatement {int i;(*f2py_func)(&n,a,&n,ipiv,jpiv,&info);for(i=0;i\<n;--ipiv[i],--jpiv[i++]);}
+    callprotoargument int*,<ctype>*,int*,int*,int*,int*
+
+    integer depend(a),intent(hide):: n = shape(a,0)
+    <ftype> dimension(n,n),intent(in,out,copy,out=lu) :: a
+    check(shape(a,0)==shape(a,1)) :: a
+    integer dimension(n),depend(n),intent(out) :: ipiv
+    integer dimension(n),depend(n),intent(out) :: jpiv
+    integer intent(out):: info
+
+end subroutine <prefix>getc2
+
+subroutine <prefix>gesc2(n,lu,rhs,ipiv,jpiv,scale)
+    ! x,scale = gesc2(lu,rhs,ipiv,jpiv,overwrite_rhs=0)
+    ! Solve  A * X = scale * RHS
+    ! A = P * L * U * Q
+    threadsafe
+    callstatement {int i;for(i=0;i\<n;++ipiv[i],++jpiv[i++]);(*f2py_func)(&n,lu,&n,rhs,ipiv,jpiv,&scale);for(i=0;i\<n;--ipiv[i],--jpiv[i++]);}
+    callprotoargument int*,<ctype>*,int*,<ctype>*,int*,int*,<ctypereal>*
+
+    integer depend(lu),intent(hide):: n = shape(lu,0)
```

Comment:
You are missing `a` and `lda` on this one.

Reply:
Similarly, `?getrs` didn't use `lda`, and I followed suit. And since `getrs` named `a`->`lu`, I did the same.